### PR TITLE
fix：KubeArmor running in host mode is dropping events due to busy event channel

### LINKED ISSUE #1155: Bug: KubeArmor running in host mode is dropping events due to busy even channel
## Bug Report

**General Information**

- Environment description: Host mode with -enableKubeArmorHostPolicy and -enableKubeArmorPolicy
- Kernel version (run `uname -a`): 5.15.81-1-MANJARO

**To Reproduce**

1. `make run-container` 
2. Wait for a minute or two and observe the KubeArmor logs

**Observed behavior**
Getting the following warning:

```
WARN	Event droped due to busy event channel
```

### PR DESCRIPTION
**Pull request by zhy76**
**Purpose of PR?**:
Bugfix: KubeArmor running in host mode is dropping events due to busy event channel
Fixes #1155
**Does this PR introduce a breaking change?**

**If the changes in this PR are manually verified, list down the scenarios covered:**:

**Additional information for reviewer?** :
_Mention if this PR is part of any design or a continuation of previous PRs_

**Checklist:**
- [x] Bug fix. Fixes #<issue number>
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update
- [x] PR Title follows the convention of  `<type>(<scope>): <subject>`
- [ ] Commit has unit tests
- [ ] Commit has integration tests

### CODE REVIEW

**Review by nyrahul — APPROVED** [review]
LGTM 👍

**Comment by zhy76** [comment]
image
What does this mean?

**Comment by daemon1024** [comment]
We have an issue with flaky tests, I will help them pass it. @zhy76

**Comment by zhy76** [comment]
PTAL

**Comment by zhy76** [comment]
any update on this pr?

**Comment by daemon1024** [comment]
Will merge #1170 soon, we can rebase this PR then and go ahead with things. Thanks for checking up. Appreciate your patience.

**Comment by zhy76** [comment]
> Will merge #1170 soon, we can rebase this PR then and go ahead with things. Thanks for checking up. Appreciate your patience.

Thanks for your reply! That will be great!

**Comment by daemon1024** [comment]
@zhy76 Can you please rebase the PR? The tests are mostly fixed now.

**Comment by zhy76** [comment]
> @zhy76 Can you please rebase the PR? The tests are mostly fixed now.

OK!

File: KubeArmor/monitor/systemMonitor.go [hunk 1]
```diff
@@ -629,10 +629,11 @@ func (mon *SystemMonitor) TraceSyscall() {
 
 			// Best effort replay
 			go func() {
-				time.Sleep(1 * time.Second)
```
Comment (achrefbensaad) [inline KubeArmor/monitor/systemMonitor.go]: Lets keep this sleep as it is needed for container logic.

**Review by achrefbensaad — CHANGES_REQUESTED** [review]

File: KubeArmor/monitor/systemMonitor.go (hunk 1 above)
Comment (zhy76) [reply KubeArmor/monitor/systemMonitor.go]: OK！

**Review by daemon1024 — APPROVED** [review]

**Review by achrefbensaad — APPROVED** [review]
LGTM! Thank you for your contribution

**Pull request merged**